kobo/apps/help/migrations/0001_initial.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
```

Comment:
@jnm: Not a bug but as we quickly discussed on flowdock.
I know this file is generated by django but: 

We can start to use PEP-8 recommendations for imports.
https://www.python.org/dev/peps/pep-0008/#imports

> Imports should be grouped in the following order:

> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.

And I would sort them alphabatically. I think it will be easier to read when we upgrade to python3

Reply:
Sure, but these migration files are automatically generated by Django. Maybe a newer version of Django follows the PEP-8 recommendations?